### PR TITLE
Improvements for german translation of Git.

### PR DESCRIPTION
**Pull request by rotespferd**
I made some improvements to the german translation of the Git document.
I also add some more example commands.

### CODE REVIEW

File: de-de/git-de.html.markdown [hunk 1]
```diff
@@ -180,6 +181,8 @@ Bringt alle Dateien im Arbeitsverzeichnis auf den Stand des Index oder des angeg
 ```bash
 # Ein Repo auschecken - wenn nicht anders angegeben ist das der master
 $ git checkout
+# Eine Datei auschecken - sie befindet sich dann auf dem aktuellen Stand im Repository
```
Comment (guntbert) [inline de-de/git-de.html.markdown]: not sure if checking out one file is really needed - or goes too deep - is it present in the English file?

File: de-de/git-de.html.markdown [hunk 2]
```diff
@@ -180,6 +181,8 @@ Bringt alle Dateien im Arbeitsverzeichnis auf den Stand des Index oder des angeg
...
+$ gut checkout /path/to/file
```
Comment (guntbert) [inline de-de/git-de.html.markdown]: TYPO!!  s/gut/git/

File: de-de/git-de.html.markdown (hunk 1 above)
Comment (rotespferd) [inline de-de/git-de.html.markdown]: I use this very often in my day-to-day work. When you edit I file and then recognize that this change was wrong or produces an error, I want to "revert" the file.

File: de-de/git-de.html.markdown (hunk 2 above)
Comment (rotespferd) [inline de-de/git-de.html.markdown]: Thanks for the hint, it is fixed now.

**Comment by vendethiel** [comment]
Thanks! :ship:

**Pull request merged**